app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
```

Comment:
I was thinking vendors == manufacturers. I think we need a `css/services/` folders for these styles.